DEVELOPMENT.md

```diff
@@ -54,51 +54,371 @@ Annotations for queries, mutations, types and fields also get propagated from
 the proxy source into the `Docs` section.
 
 
-## Releases
+## UI
 
-To perform a release:
+For dependency management and script execution we use `yarn`. Code formatting
+is dictated by [prettier][pr] and enforced locally on a pre-commit basis with
+[husky][hu] and [lint-staged][ls].
 
-1. If you haven't already, install the [`hub`][hub] cli tool.
 
-2. Run: `cd app && yarn release` and follow the instructions.
-   Here's what a typical release looks like:
+### Running the app
+
+1. Get the code: `git clone git@github.com:radicle-dev/radicle-upstream.git`.
+2. Install dependencies: `cd radicle-upstream && yarn install`.
+3. Start the app in development mode: `yarn start`.
+
+
+### Running tests
+
+Before running tests locally you'll need to set up a test fixture repository:
+`git submodule foreach "git fetch --all"`.
```

Comment:
Does this work without a `git submodule update --init --remote` before?

Reply:
Nope, we need to init submodules before. Will add.

Reply:
It does not.